html-links/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Popular Search Engines</title>
+    <h1>Popular Search Engines</h1>
+  </head>
+  <body>
+    <ul>
+      <li><a href="http://www.google.com">Google</a></li>
+      <li><a href="http://www.yahoo.com">Yahoo</a></li>
+      <li><a href="http://www.bing.com">Bing</a></li>
+      <li><a href="Ask">Ask</a></li>
```

Comment:
This link wouldn't work 🤔

Reply:
Hi @codingcodymiller I added a valid Ask.com link!

<img width="785" alt="Screen Shot 2021-11-30 at 12 53 21 PM" src="https://user-images.githubusercontent.com/76764048/144126542-293f7ed6-d4fc-4584-9a87-e742381650fd.png">
: